greedybear/settings.py

```diff
@@ -15,6 +15,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = os.environ.get("DEBUG", False) == "True"
 
+# DB Backup Configs
+DBBACKUP_STORAGE = "django.core.files.storage.FileSystemStorage"
+DBBACKUP_STORAGE_OPTIONS = {"location": "/var/log/greedybear/"}
```

Comment:
these are not logs, I would leverage a new directory like `/opt/backups`

Reply:
we also need to provide a mechanism for deletion of too old files in this directory.

I would say a maximum of 10 backups we keep there as default and then we delete the old ones

Reply:
I've fixed that. Please check